packages/governance-modules/cannonfile.toml

```diff
@@ -0,0 +1,92 @@
+name = "synthetix-governance"
+description = "Governance cotracts for on chain voting"
+version = "0.0.1"
+
+# Deployment Owner, defaults to first hardhat account
+[setting.owner_address]
+defaultValue = "0xe3B0C44298FC1C149AfBF4C8996fb92427aE41E4"
+
+[setting.council_token_name]
+defaultValue = "Synthetix Governance Module"
+
+[setting.council_token_symbol]
+defaultValue = "SGT"
+
+[setting.first_council_member]
+defaultValue = "0xe3B0C44298FC1C149AfBF4C8996fb92427aE41E4"
+
+[setting.epoch_duration]
+defaultValue = 90 # days
+
+[setting.voting_period_duration]
+defaultValue = 7 # days
+
+[contract.election_module]
+artifact = "contracts/modules/ElectionModule.sol:ElectionModule"
+step = 1
+
+[contract.owner_module]
+artifact = "contracts/modules/OwnerModule.sol:OwnerModule"
+step = 1
+
+[contract.upgrade_module]
+artifact = "contracts/modules/UpgradeModule.sol:UpgradeModule"
+step = 1
+
+[run.generate_router]
+exec = "scripts/generate-router.js"
+func = "generateRouter"
+args = [
+  "contracts/modules/ElectionModule.sol:ElectionModule",
+  "<%= outputs.self.contracts.election_module.address %>",
```

Comment:
srsly this works? again, great job!

Reply:
wonder if we could mak ethe params even more streamlined though. thoughts?

Reply:
hmm, my original idea was to only give it 'outputs.self.contracts', so we would avoid repetition, but we cannot do it because cannon doesn't allow us to put objects thru templating. Maybe we could ask the mainteners for that feature? 😬

Reply:
hmm maybe we should ask :thinking:

Reply:
Ended up using a literal JSON on a multi line string, I think it works great :)